frontend/src/components/dashboard/tips/Tips.module.scss

```diff
@@ -52,8 +52,6 @@
   background-color: $color-white;
   width: $content-sm;
   max-width: calc(100% - $spacing-xl);
-  // Prevent the BlockLevelSlider thumb, which is also absolutely positioned
```

Comment:
I could remove this because I added `isolation: isolate` to `profile.module.scss`.